indico/modules/events/persons/schemas.py

```diff
@@ -5,17 +5,61 @@
 # modify it under the terms of the MIT License; see the
 # LICENSE file for more details.
 
-from marshmallow import fields
+from marshmallow import EXCLUDE, fields, post_dump
 
 from indico.core.marshmallow import mm
 from indico.modules.events.models.persons import EventPerson
+from indico.modules.users.models.users import UserTitle
+from indico.util.i18n import orig_string
+
+
+class PersonLinkSchema(mm.Schema):
+    class Meta:
+        unknown = EXCLUDE
+
+    _type = fields.Constant('PersonLink')
```

Comment:
Does this need to interact with any legacy code that expects `_type`? If not let's maybe use just `type`